pom.xml

```diff
@@ -1338,6 +1338,11 @@
           <artifactId>wildfly-maven-plugin</artifactId>
           <version>1.2.0.Final</version>
         </plugin>
+        <plugin>
```

Comment:
Actually, I compiled ```drools-wb``` locally (with your PR and without this PR) and it succeeded. 

So, on investigation, I found [this](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L956). Do we need to re-declare the plugin or simply add the version to the existing entry?

Reply:
@manstis I will add the version to the existing entry and remove the new declaration.
Fixed with
[DROOLS-2976] Add version to existing build-helper-maven-plugin declaration. Removing duplicated one